tests/integration/test_static_analysis_consistency.py

```
@@ -137,8 +137,10 @@ def _write_snapshot(static_analysis: StaticAnalysisResults, language: str, confi
     return snapshot_path
 
 
-# Tolerance percentage for metric comparisons (2.5% = 0.025)
-METRIC_TOLERANCE = 0.025
+# Tolerance for metric vs fixture (relative diff). Raised from 2% so large PHP
+# graphs (e.g. wordpress_php call_graph_nodes) stay stable on Windows where
+# intelephense/LSP variance can reach ~2.5–2.6% (e.g. 499/19646 ≈ 2.54%).
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)